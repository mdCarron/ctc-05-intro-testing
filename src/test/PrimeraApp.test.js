import React from "react";
import PrimeraApp from "../PrimeraApp";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

describe("Testing en PrimeraApp", () => {
  /*   test("should mostrar mensaje Hola, soy un Titlo", () => {
    const saludo = "Hola, soy un Titulo";
    const { getByText } = render(<PrimeraApp saludo={saludo} />);

    expect(getByText(saludo)).toBeInTheDocument();
  }); */
  test("debe de mostrar <PrimeraApp /> correctamente", () => {
    const saludo = "Hola, soy un Titulo";
    const wrapper = shallow(<PrimeraApp saludo={saludo} />);

    expect(wrapper).toMatchSnapshot();
  });
});
