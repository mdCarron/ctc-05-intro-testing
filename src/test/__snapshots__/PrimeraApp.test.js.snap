// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing en PrimeraApp debe de mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, soy un Titulo
  </h1>
  <p>
    Este es el subtitulo
  </p>
</Fragment>
`;
